src/LibreCards.Core/ILobby.cs

```diff
@@ -11,7 +11,7 @@ public interface ILobby
         void RemovePlayer(Guid id);
         Player GetPlayer(Guid id);
         int PlayerCount { get; }
-        IEnumerable<Player> GetPlayers();
+        IEnumerable<Player> Players();
```

Comment:
Heya,
Thank you so much for being willing to help us out, it means a lot. 😊 

I probably didn't explain it well enough, but the idea would be to change this into a property.

In C# [properties](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties) are a nice way of hiding get and set methods.

So instead of writing something like
```cs
Data GetUserData();
```

we can use a property like this:

```cs
Data UserData { get; }
```

😎 

The major difference between them is how they're then called.
```diff
- var players = lobby.GetPlayers(); // old, method
+ var players = lobby.Players; // new, property
```

I hope it makes sense. 😅 

So I'd suggest the following edit:

```suggestion
        IEnumerable<Player> Players { get; }
```

---

There are other minor changes required after that, for example since we changed the name of `GetPlayers()` we're going to have to also change it [here](https://github.com/petrspelos/libre-cards-against-humanity/blob/main/src/LibreCards.Core/Game.cs#L57), [here](https://github.com/petrspelos/libre-cards-against-humanity/blob/main/src/LibreCards.Core/Game.cs#L47), and [here](https://github.com/petrspelos/libre-cards-against-humanity/blob/main/src/LibreCards.Core/Lobby.cs#L38).

Of course, I don't want to scare you away from your first contribution, so it's perfectly fine if you only apply the suggestion from earlier, we can then fix it on all the mentioned places later. ☺️

Reply:
Hi @petrspelos , Thank you so much for explaining this whole concepts so nicely. I have tried to modified things as you mentioned, Please correct me if I am wrong anywhere. Once again Thank you for making me comfortable with open source :)